### ISSUE TITLE
Space between sum & currency symbol

### ISSUE THREAD

**Issue by Cantibra** [crater-invoice/crater#881]
**Describe the bug**
There is no space between the sum and the currency symbol on all createable PDF's.

**Expected behavior**
A space between the sum and the currency symbol or at least a variable.

**Screenshots**
![Bildschirmfoto 2022-04-06 um 12 44 01](https://user-images.githubusercontent.com/87247206/161957973-67236f7e-555b-46af-b119-2bdf7adedc8a.png)

**Please complete the following information:**
- Crater version: 6.0.6
- PHP version: 7.4.28
- Database type and version: MariaDB (SQL) - 10.5.15

**Comment by enoversum** [comment]
This seems to be somewhat hardcoded into the core or a function. Also, the currency sign shows bigger in my template while using a custom font, I assume because it uses the standard font?

**Comment by gnovaro** [comment]
There is a helper for format money inside templates, i think should be have a new parameter for format should be setting across config in my opion like he does in line 133, also have a harcode font, this should be in a css file with a class
https://github.com/crater-invoice/crater/blob/eea3925fcdafb60099182194b119fc5836cfa6da/app/Space/helpers.php#L128

**Comment by enoversum** [comment]
> There is a helper for format money inside templates, i think should be have a new parameter for format should be setting across config in my opion like he does in line 133, also have a harcode font, this should be in a css file with a class
> 
> https://github.com/crater-invoice/crater/blob/eea3925fcdafb60099182194b119fc5836cfa6da/app/Space/helpers.php#L128

Yes, thank you @gnovaro! It's bad practice to alter system files, but for the time being, this is removing the font display problem in my templates. Now everything looks tight. I'm glad you found it!
